supervisord.conf

```diff
@@ -0,0 +1,23 @@
+[supervisord]
+#user=tango
+nodaemon=true
+logfile=/var/tmp/supervisord.log
+pidfile=/var/tmp/supervisord.pid
+
+[program:tango-starter]
+command=/bin/bash -c '
+  HOSTNAME=`hostname --short` &&
+  /usr/local/bin/tango_register_device Starter/$HOSTNAME Starter tango/admin/$HOSTNAME &&
+  /usr/local/bin/tango_admin --add-property tango/admin/$HOSTNAME StartDsPath /usr/local/bin &&
+  exec -- /usr/local/bin/Starter $HOSTNAME'
+autorestart=true
+user=tango
+priority=1
+
+[program:tango-test]
+command=/bin/bash -c '
+  /usr/local/bin/tango_admin --add-property sys/tg_test/1 VIP Whoa! &&
+  exec -- /usr/local/bin/TangoTest test'
+autorestart=true
```

Comment:
Is it a good idea to autorestart TangoTest?
What if you want to test what happens when the device server is stopped with device admin kill command?
If I understand well, supervisord will restart TangoTest/test automatically in this situation, which can be quite confusing/surprising for the user who explicitly requested the device server to be killed.
`autorestart=unexpected` might be a better choice? What do you think?

Reply:
@bourtemb , good point! I did not know about `autorestart=unexpected`, worth giving it a try.